pysmarthashtag/vehicle/basic_status.py

```
@@ -0,0 +1,152 @@
+            if config:
+                retval["propulsion_type"] = get_field_as_type(config, "propulsionType", int)
+                retval["fuel_type"] = get_field_as_type(config, "fuelType", int)
+
+            # Timestamp
+            if "updateTime" in vehicle_status:
+                retval["timestamp"] = datetime.fromtimestamp(int(vehicle_status["updateTime"]) / 1000)
+
+        except KeyError as e:
+            _LOGGER.info(f"Basic status info not available: {e}")
+        finally:
+            return retval
```

Ruff (0.15.2)

[warning] `return` inside `finally` blocks cause exceptions to be silenced

(B012)